src/foundry/foundry.js/applications/sidebarTabs/combatTracker.d.ts

```diff
@@ -142,21 +153,39 @@ declare global {
           control: false;
         }
       | {
-          combats: StoredDocument<InstanceType<ConfiguredDocumentClass<typeof Combat>>>[];
-          currentIndex: number;
+          user: Game['user'];
+          combats: CombatTracker['combats'];
           combatCount: number;
+          started: boolean;
+          settings: ClientSettings.Values[`core.${typeof Combat.CONFIG_SETTING}`];
+          currentIndex: number;
           hasCombat: true;
           combat: StoredDocument<InstanceType<ConfiguredDocumentClass<typeof Combat>>>;
           turns: Turn[];
           previousId: string | null;
           nextId: string | null;
-          round: number;
-          turn: number;
           control: boolean;
-        }
-    );
+          round: foundry.data.CombatData['round'];
+          turn: foundry.data.CombatData['turn'];
+        };
+
+    type Turn = {
+      id: StoredDocument<InstanceType<ConfiguredDocumentClass<typeof Combatant>>>['id'];
```

Comment:
this seems over complicated, maybe just make it `string`? Similar for the others. I don't see much value in typing these dependent on those classes, so it just unnecessarily drives compile times up